client/src/axiom/utils.ts

```diff
@@ -5,10 +5,23 @@ import {
   AxiomV2FeeData,
   getByteLength,
 } from "@axiom-crypto/core";
-import { AxiomV2SendQueryArgsParams, CircuitInputType } from "../types";
+import { AbiType, AxiomV2ClientOverrides, AxiomV2SendQueryArgsParams, CircuitInputType } from "../types";
 import { createPublicClient, http } from 'viem';
 import { mainnet, sepolia } from 'viem/chains';
 import { ClientConstants } from "../constants";
+import { getAxiomV2Abi, getAxiomV2QueryAddress, viemChain } from "../lib";
+
+export function validateChainId(chainId: string) {
+  switch (chainId) {
+    case "1": // Mainnet
+    case "8453":  // Base
```

Comment:
Remove Base and Base Sepolia for now; this will better test the override feature